### DISCUSSION TITLE
Account suspended (blocked) - Unable to login

### DISCUSSION THREAD

**Discussion by lightla** [community/community#145079] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### GitHub Feature Area

Other

### Body

I want my code, if not recover my account, please give me code @github-staff 

<Personal into removed by Staff>

**Comment by Awwze** [comment]
Hi light-developer,

If your account has been suspended and you cannot log in, here’s what you can do to recover your code or your account:

Check Your Email:

GitHub usually sends an email explaining why your account was suspended. Be sure to review your inbox (and spam folder) for any messages from GitHub Support. This email may contain instructions on how to appeal the suspension.
Contact GitHub Support:

To resolve this issue, you’ll need to reach out directly to GitHub Support. Use the following link: [GitHub Support Contact](https://support.github.com/contact).
Provide as many details as possible, including your GitHub username, the nature of your issue, and any relevant context. If you want your code, explicitly mention that in your request.

GitHub’s support team is usually prompt, so reaching out to them is your best course of action. Hope this gets resolved for you soon!
